src/Installer/finalizer/finalizer.csproj

```
@@ -0,0 +1,30 @@
+﻿<Project Sdk="Microsoft.NET.Sdk">
+
+  <PropertyGroup>
+    <TargetFramework>$(SdkTargetFramework)-windows</TargetFramework>
+    <OutputType>WinExe</OutputType>
+    <Nullable>enable</Nullable>
+  </PropertyGroup>
+
+  <PropertyGroup Condition="$(NETCoreSdkPortableRuntimeIdentifier.StartsWith('win-'))">
+    <PublishAot>true</PublishAot>
```

The native EXE had `/guard:cf` in the CMakeLists.txt, this program will likely get flagged by security compliance tooling without this:

```suggestion
    <PublishAot>true</PublishAot>
    <ControlFlowGuard>Guard</ControlFlowGuard>
```